apps/okular/okular.py

```diff
@@ -0,0 +1,43 @@
+from talon import Module, Context, actions
+
+# --- App definition ---
+mod = Module()
+mod.apps.okular = """
+os: windows
+and app.name: okular.exe
+os: windows
+and app.exe: okular.exe
+"""
+mod.apps.okular = """
+os: linux
+and app.name: okular
+"""
+# TODO: mac context and implementation
+
+# Context matching
+ctx = Context()
+ctx.matches = """
+os: windows
+os: linux
+app: okular
```

Comment:
As above, don't you need to do
```suggestion
os: windows
and app: okular
os: linux
and app: okular
```

Reply:
See above